app/src/main/res/layout/session_list_item.xml

```diff
@@ -67,14 +65,13 @@
     <ImageView
         android:id="@+id/session_list_item_without_video_recording_view"
         android:layout_width="wrap_content"
-        android:layout_height="14sp"
+        android:layout_height="wrap_content"
```

Comment:
Why is this icon set to `wrap_content` while the others are set to `24dp`?

Reply:
Just forgott to change that :sweat_smile:

Reply:
I realized that I actually should have changed the others to, so now they all are at `24.15sp`, so they scale with the font size :+1: